src/schemas/json/hammerkit.json

```
@@ -0,0 +1,349 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "definitions": {
+    "labels": {
+      "description": "A map of label values",
+      "type": "object",
+      "additionalProperties": { "type": "string" }
+    },
+    "envMap": {
+      "description": "A map of environment values",
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "taskCommand": {
+      "anyOf": [
+        {
+          "description": "Command to be executed for the task",
+          "type": "string"
+        },
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "cmd": {
+              "description": "Command to be executed for the task",
+              "type": "string"
+            },
+            "path": {
+              "description": "Path below build file path",
+              "type": "string"
+            }
+          },
+          "required": ["cmd"]
+        }
+      ]
+    },
+    "include": {
+      "type": "object",
+      "description": "A map of includes",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "reference": {
+      "type": "object",
+      "description": "A map of references",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "taskSource": {
+      "type": "array",
+      "description": "A list of source files and folders",
+      "items": {
+        "type": "string"
+      }
+    },
```

```suggestion
    "taskSource": {
      "description": "Source files and folders for the current project",
      "type": "array",
      "uniqueItems": true,
      "items": {
        "$ref": "https://json.schemastore.org/base.json#/definitions/path"
      }
    },
```

Almost the same issue as above.
Optional fix: `description` can be simplified.
Optional fix: unexpected `items` property, `uniqueItems` expected.
Mandatory fix: [unexpected `string` type, `$ref` to `base.json` expected](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices).